frontend/integration-tests/views/crud.view.ts

```diff
@@ -1,6 +1,6 @@
 import * as _ from 'lodash';
 import { safeDump, safeLoad } from 'js-yaml';
-import { $, $$, browser, by, ExpectedConditions as until, element } from 'protractor';
+import { $, $$, browser, by, ExpectedConditions as until } from 'protractor';
```

Comment:
Just want to confirm if "element" is still needed by Line 31 ?
 export const navTabFor = (name: string) => navMenu.element(by.linkText(name));

Reply:
It's not needed since it's a property on `navMenu`, which is different